docs/man/rpm-queryformat.7.scd

```diff
@@ -0,0 +1,290 @@
+RPM-QUERYFORMAT(7)
+
+# NAME
+*rpm-queryformat* - RPM query format
+
+# SYNOPSIS
+
+%[_WIDTH_]{_TAG_[:_FORMAT_]}
+
+# DESCRIPTION
+
+The *rpm* query format is a static string modeled after the standard *printf*(3)
```

Comment:
I know this comes from the original text, but while we're at it: I'd take out these printf(3) mentions, because in reality it's almost nothing like the printf format. The only real resemblance is the width field, where it actually is appropriate. The term "static string" seems also a bit strange  - if anything it's a dynamically interpreted string.